### PR TITLE
822: Update data service for LDU caseload

### PR DESCRIPTION
**Pull request by caoimheb**
Replace get-team-caseload with a more generic get-caseload. Add caseloadView
to the org unit constant to allow query to call the correct view.

Add integration tests for LDU level caseload. Update data helper to insert
offender managers of both PO and PSO, for more realistic tests

### CODE REVIEW

File: app/services/data/get-caseload.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  if (orgUnit.name === orgUnitConstants.TEAM.name) {
+    selectColumns.push('case_type AS caseType')
+  }
+
+  return knex(table)
+    .where('id', id)
+    .select(selectColumns)
+    .then(function (results) {
+      return results
+    })
```
Comment (dconey646) [inline app/services/data/get-caseload.js]: 👍

File: package.json [hunk 2]
```diff
@@ -7,7 +7,7 @@
     "start": "gulp",
     "build": "yarn install && gulp generate-assets && gulp sync",
     "unit-test": "standard && mocha --recursive test/unit/",
-    "integration-test": "gulp assets && mocha --recursive test/integration --timeout=10000",
+    "integration-test": "gulp assets && mocha --recursive test/integration --timeout=30000",
```
Comment (dconey646) [inline package.json]: How come an increase in the timeout?

File: test/helpers/data/aggregated-data-helper.js [hunk 3]
```diff
@@ -119,93 +141,116 @@ var addLdu = function (inserts) {
...
-      })
-      return inserts
+    return knex('tiers').returning('id').insert(tiers)
+  })
+  .then(function (ids) {
+    ids.forEach((id) => {
+      inserts.push({table: 'tiers', id: id})
     })
+    return inserts
+  })
```
Comment (dconey646) [inline test/helpers/data/aggregated-data-helper.js]: 👍

File: package.json (hunk 2 above)
Comment (caoimheb) [inline package.json]: The data inserts and removes were timing out on me. Not sure if it's maybe because I'm on a worse network when WFH or something.

**Pull request merged**